packages/network/lib/uri.ts

```diff
@@ -6,7 +6,7 @@
 // - https://nodejs.org/api/url.html#url_url_format_urlobject
 
 import _ from 'lodash'
-import url from 'url'
+import url, { URL } from 'url'
```

Comment:
This seems like a typo.

Reply:
Yeah... `url` is the [legacy API](https://nodejs.org/api/url.html#legacy-url-api) where `URL` is the [latest and greatest](https://nodejs.org/api/url.html#url). Probably should refactor all the `url` parts to use `URL` instead, but that's out of scope for this work.